force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -227,19 +264,25 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Book {
         // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+        public String title;
+        public String author;
+        public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String title, String author) {
             // Initialize the title and author
+            this.title = title;
+            this.author = author; 
         }
-    
+        //I create a setter to manipulate a pages in Question 13;
+        // void setNumbersOfPages(String pages){
+        //     this.pages = pages;
+        // }
+
         // Method that returns the details of the book
         public String getBookDetails() {
             // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            return ('Title: '+ title +', Author: '+author); 
```

Comment:
```suggestion
            return ('Title: '+ this.title +', Author: '+ this.author); 
```
Althought the keyword is not required, it's best practice to use the 'this' keyword